src/modules/assets/copy/copy.ts

```diff
@@ -0,0 +1,43 @@
+import { type IOdsModulesCopy } from './IOdsModulesCopy';
+
+export const enCopy: IOdsModulesCopy = {
+    addressInput: {
+        clear: 'Clear',
+        paste: 'Paste',
+    },
+    assetDataListItemStructure: {
+        unknown: 'Unknown',
+    },
+    memberDataListItemStructure: {
+        yourDelegate: 'Your Delegate',
+        you: 'You',
+        delegations: 'Delegations',
+        votingPower: 'Voting Power',
+    },
+    approvalThresholdResult: {
+        stage: 'Stage',
+        of: 'of',
+        members: 'Members',
+    },
+    majorityVotingResult: {
+        winningOption: 'Winning Option',
+        stage: 'Stage',
+    },
+    proposalDataListItemStatus: {
+        youHaveVoted: "You've voted",
```

Comment:
In that way, if we later change it to `you did vote` or `voted successfully`, the label still makes sense
```suggestion
        voted: "You've voted",
```

Reply:
✅ https://github.com/aragon/ods/pull/221/commits/e0774462dbc0f20790ca7353858bf1aaeee95943